libraries/AP_Mount/AP_Mount_Backend.cpp

```diff
@@ -161,3 +171,12 @@ void AP_Mount_Backend::calc_angle_to_location(const struct Location &target, Vec
         }
     }
 }
+
+// calc_angle_to_location - calculates the earth-frame roll, tilt and pan angles (and radians) to point at the given sysid
+void AP_Mount_Backend::calc_angle_to_sysid(const struct Location &target, Vector3f& angles_to_target_rad, bool calc_tilt, bool calc_pan, bool relative_pan)
```

Comment:
it's a bit funny that the method's name is calc-angle-to-sysid but no sysid is specified.. only a Location is specified.  We should probably fix the comments above the method in any case.

Reply:
ah.. and stranger still, the calc-angle-to-location doesn't take a location but rather a sysid.

Reply:
the calc-angle-to-sysid was a dead method.

The call to calc-angle-to-location is clarified with the variable renames.